src/node/blockstorage.h

```diff
@@ -120,6 +142,16 @@ class BlockManager
 
     CBlockIndex* LookupBlockIndex(const uint256& hash) const EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+    /** Get block file info entry for one block file */
```

Comment:
Would be it worth mentioning that this function is only used by the unit tests? If yes, can add to #24002. 

```diff
-    /** Get block file info entry for one block file */
+    /** Get block file info entry for one block file. Currently only used for unit tests. */
```

Reply:
It might be better to move the function to the test, if it is only used there.